.travis.yml

```diff
@@ -39,6 +40,8 @@ before_script:
   - echo $LIBCLANG_PATH
 
 script:
+  - export PATH=$PATH:~/.cargo/bin
+  - cargo fmt -- --write-mode=diff
```

Comment:
Soft -1 if this is going to cause travis CI to fail if there is any difference in the formatting. As mentioned before, we will hit this issue with different `rustfmt` versions.